textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/constants/TestConstants.java

```diff
@@ -65,4 +65,52 @@
         return Arrays.asList(tuple1, tuple2, tuple3, tuple4, tuple5);
                 
     }
+    
+    //sample created by team3

```

Comment:
@zuozhi .. But as you said, it is always better to include the test related classes in src/test/java. So, if you are creating a new schema and it is being used only in that project, it is better to create a new Constants class (TestXXXXConstants ) and include that in src/test/java folder